.gitignore

```diff
@@ -18,4 +18,4 @@ yarn-error.log*
 
 # Sistema operativo
 .DS_Store
-Thumbs.db
+Thumbs.db
```

Comment:
Que añadáis "ruido" a un PR haciendo estos cambios no acaba de parecer bien. Si vais a añadir algo, que sea un .gitignore del lenguaje con el que estéis trabajando.

Reply:
Hice ese cambio por error al reiniciar la rama desde 0. Eso lo añadió @JLombar en el objetivo 1. Añadiré lo referente a Python.